networking/bgp.md

```diff
@@ -140,6 +141,17 @@ spec:
   peerSelector: route-reflector == 'true'
 ```
 
+Last, you will need to disable the node-to-node mesh, e.g. by configuring
```

Comment:
Sadly, looks like a dupe of the section a few above this: https://github.com/projectcalico/calico/blob/f623f5f3b7cfe85fdefe3881796f344c7e0cc91a/networking/bgp.md#disable-the-default-bgp-node-to-node-mesh

Reply:
Hmm, that is true, that is the place from where I got the info what to fix. Maybe the problem is more the structure of the document. The first item (chaging the nodeToModeMeshEnabled setting, it relevant for all of the following. But then there are 3 different options (external peer, ToR, route reflector). The section about reflector (at least for me) suggested, that that is all which needs to be done and all of the above is only for the other setups. 

So maybe it should be more like:
* HowTo
  * use different BGP topologies then node2node
    * Disable the default BGP node-to-node mesh
    * Configure one of these topologies
      * global BGP peer
      * per-node BGP peer
      * node to act as a route reflector
  * View BGP peering status for a node